tests/bdd/features/volume/topology/node-topology.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Volume Node Topology
+  The volume topology feature is for selection of particular nodes to schedule a replica on.
+  All the volume topology key and values should have an exact match or the keys must match with node labels, although nodes
+  can have extra labels, for the selection of that node. The affinity topology label implies the inclusion label 
+  for node with both key and value or just keys. The storage class has the labels as
+    nodeTopologyAffinity: |
+     A1: B1
+     C E : D1
+    which signifies the volume node topology inclusion labels as {"A1": "B1", "C E" : "D1"}
+
+    nodeHasTopologyKey: |
+      alpha
+      beta    
+   which signifies the volume node topology inclusion labels as {"alpha": "", "beta" : ""} 
+
+  Background:
+    Given a control plane, four Io-Engine instances, twelve pools
+
+
+# The labels to be applied to the nodes.
+#################################################################################################
+#   Description              ||  Node Name             ||    Pool Name   ||     Node Label     ||
+#==============================================================================================
+#     "io-engine-1" has      ||   node1 (io-engine-1)  ||    node1pool1  ||                    ||
+#  label "alpha1=beta1"      ||   node1 (io-engine-1)  ||    node1pool2  ||     alpha1=beta1   ||
```

Comment:
nit: would be nice to use rack label as the example (eg: rack=1,zone=a, rather than alpha=beta) so more akin to real life usage